src/commands/deploy/spec/cbd.spec.server.ts

```diff
@@ -237,7 +237,7 @@ describe('sasjs cbd having stream app', () => {
 
       await updateLocalTarget(target.name, { appLoc })
     },
-    2 * 60 * 1000
+    5 * 60 * 1000
```

Comment:
Why you've decided to increase the timeout?

Reply:
Last commit is actually about this, server is taking too long for these 2 tests locally and in [github ci](https://github.com/sasjs/cli/runs/3137496360)

You may have look at other current PR's [actions](https://github.com/sasjs/cli/pull/855/checks?check_run_id=3138966454)

Reply:
we can just use the default timeout set in `jest.server.config.js`

Reply:
updated